tests/std/tests/GH_000814_avx512_vector_algorithms/env.lst

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\prefix.lst
+RUNALL_CROSSLIST
+PM_CL="/EHsc /MT /O2 /GL /arch:AVX512 /std:c++latest /permissive- /analyze:autolog- /w14640 /Zc:threadSafeInit-"

```

Comment:
In that case, this flag may not be necessary. From the parameter [documentation](https://docs.microsoft.com/en-us/cpp/build/reference/arch-x64?view=msvc-160):

> /arch:AVX512
   Enables the use of Intel Advanced Vector Extensions 512 instructions.

That sounds to me like this option may be necessary to use AVX-512.

The test is written to fail if AVX-512 instructions are not supported. This may cause failure in any environment running the test suite without AVX-512 support, especially ARM(#813). In this case, it is required to ensure this test runs and only runs on platforms that are able to test. I don't see any way to express this using the testing framework. Therefore I will change the default behavior to succeed if AVX-512 is not supported at runtime or by architecture and add `arch:AVX512` to the default flags of the test suite, according to your suggestion.